File: 프로그래머스/0/120854. 배열 원소의 길이/배열 원소의 길이.py
def solution(strlist):
    # 각 문자열의 길이를 저장할 리스트 초기화
    answer = []

    # 문자열 리스트를 순회하면서 각 문자열의 길이를 구함
    for s in strlist:
        # 현재 문자열의 길이를 리스트에 추가
        answer.append(len(s))

    # 최종적으로 길이 리스트 반환
    return answer

# ✅ 예제 테스트 실행
print(solution(["We", "are", "the", "world!"]))
# 결과: [2, 3, 3, 6] (각 단어의 길이를 리스트로 반환)

print(solution(["I", "Love", "Programmers."]))
# 결과: [1, 4, 12] (각 단어의 길이를 리스트로 반환)

File: 프로그래머스/0/120854. 배열 원소의 길이/배열 원소의 길이.java
public class Solution {

    public static int[] solution(String[] strlist) {
        int[] answer = new int[strlist.length];

        // 각 문자열의 길이를 배열에 저장
        for (int i = 0; i < strlist.length; i++) {
            answer[i] = strlist[i].length();
        }

        return answer;
    }

    // ✅ 테스트 실행
    public static void main(String[] args) {
        String[] input1 = {"We", "are", "the", "world!"};
        String[] input2 = {"I", "Love", "Programmers."};

        System.out.println(java.util.Arrays.toString(solution(input1))); // [2, 3, 3, 6]
        System.out.println(java.util.Arrays.toString(solution(input2))); // [1, 4, 12]
    }
}
